### DISCUSSION TITLE
Is react-icons dead?

### DISCUSSION THREAD

**Discussion by jonathanstanley** [react-icons/react-icons#533] [Q&A] [answered]
I like the concept and added this package today but quickly found an issue with missing icons.  But in checking,  I noticed this repo has 140 issues, 28 PRs, but only a dozen or so commits in the last year.  I can't find any other indication on the status, so just curious what the direction will be here?

**Comment by ricardo-valero** [comment]
I don't know been looking for alternatives, here's one I'm currently using
https://github.com/antfu/unplugin-icons

**Reply by jonathanstanley** [reply]
I also found another that has worked well: https://iconify.design/ (also not limited to react)

**Comment by nolanleung** [comment, marked as answer]
Hey! Currently working on a re-write to support webpack 5

**Comment by lofi-marz** [comment]
Any update on this? ~5 months later and the repo seems to be in the same state, plenty of open issues and unmerged PRs
